### PR TITLE
Add command to disable application quarantine msg

### PR DESCRIPTION
**Pull request by issmirnov**
Source: https://ss64.com/osx/syntax-defaults.html

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -667,3 +667,16 @@ categories:
...
+      - key: LSQuarantine
+        domain: com.apple.LaunchServices
+        title: Disable application quarantine message
+        description: Turn off the “Application Downloaded from Internet” quarantine warning
+        param:
+          type: bool
+        examples:
+          - value: true
+            default: true
+            text: "Application Downloaded from Internet" popup will display
```
Comment (yannbertrand) [inline defaults.yml:679]:
```suggestion
            text: '"Application Downloaded from Internet" popup will display'
```

File: defaults.yml [hunk 2]
```diff
@@ -667,3 +667,16 @@ categories:
...
+          - value: false
+            text: "Application Downloaded from Internet" popup will not display
```
Comment (yannbertrand) [inline defaults.yml:681]:
```suggestion
            text: '"Application Downloaded from Internet" popup will not display'
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]
I tried it on Big Sur, I'm not sure how to get this message?

Here is my settings:
settings

That's what happens when trying to open a newly downloaded App using Spotlight:
open-spotlight

And by right clicking it + "Open" in the Applications folder:
open-applications-folder

Does it only affect downloaded Apps? Not the ones installed through Homebrew?

**Review by yannbertrand — APPROVED** [review]

**Comment by yannbertrand** [comment]
Ok I think I found it by downloading a `.dmg` file and dragging the App to the Applications folder

**Comment by yannbertrand** [comment]
downloaded

**Comment by yannbertrand** [comment]
```
defaults write com.apple.LaunchServices LSQuarantine -bool false
```

seems not to affect it tho

**Comment by issmirnov** [comment]
You are right - I was testing on an older machine. Sorry about the confusion. Closing PR so that it doesn't clutter up the list.

**Comment by yannbertrand** [comment]
It's alright! Will just had a warning to explain it doesn't work on Big Sur.

File: defaults.yml [hunk 3]
```diff
@@ -667,3 +667,16 @@ categories:
             text: Notifications will be displayed.
         versions: [Catalina]
         after: killall Music
+      - key: LSQuarantine
+        domain: com.apple.LaunchServices
+        title: Disable application quarantine message
+        description: Turn off the “Application Downloaded from Internet” quarantine warning
```
Comment (yannbertrand) [inline defaults.yml:673]:
```suggestion
        description:
          "Turn off the “Application Downloaded from Internet” quarantine warning.<br>
          :warning: Stopped working on Big Sur."
```

File: defaults.yml [hunk 4]
```diff
@@ -667,3 +667,18 @@ categories:
             text: Notifications will be displayed.
         versions: [Catalina]
         after: killall Music
+      - key: LSQuarantine
+        domain: com.apple.LaunchServices
+        title: Disable application quarantine message
+        description:
+          "Turn off the “Application Downloaded from Internet” quarantine warning.<br>
+          :warning: Stopped working on Big Sur."
```
Comment (yannbertrand) [inline defaults.yml:675]:
```suggestion
          ⚠️ Stopped working on Big Sur."
```

**Comment by yannbertrand** [comment]
Thank you 😄

**Pull request merged**